docs/sources/setup-grafana/configure-security/configure-authentication/azuread.md

```diff
@@ -100,6 +100,30 @@ To enable the Azure AD OAuth2, register your application with Azure AD.
 
 1. Click on **Users and Groups** and add Users/Groups to the Grafana roles by using **Add User**.
```

Comment:
This step confuses me a bit. Is the user clicking **Users and Groups** and then clicking **Add User**? I'm not sure what you mean by 'using Add User'.

Reply:
this is not related to this PR and no change was applied here
.
I've checked Azure's UI and this is indeed what the author of the line meant: The user clicks Users and Groups and then clicks Add User once they arrive on the new menu